src/main/java/org/jabref/logic/importer/fileformat/MarcXmlParser.java

```diff
@@ -398,6 +404,59 @@ private void putNotes(BibEntry bibEntry, Element datafield) {
         }
     }
 
+    private void putIssue(BibEntry bibEntry, Element datafield) {
+        bibEntry.setType(StandardEntryType.Article);
+
+        List<String> issues = getSubfields("g", datafield);
+
+        for (String issue : issues) {
+            String[] parts = issue.split(":");
+            if (parts.length == 2) {
+                String key = parts[0].trim();
+                String value = parts[1].trim();
+
+                if (StringUtil.isNotBlank(value)) {
+                    switch (key) {
```

Comment:
Please use modern switch syntax without "break", which is notoriously being forgotten.
case "case" -> { /* statements here */ }

Reply:
Fixed